mfthomps/CyberCIEGE-port-to-Unity3d
# Merge Request for Issue/31 - user view

Closes #31 

To test:
- Play Introductory scenario
- Make sure you can complete it by training users on the User view at the appropriate time

Notes:
- Requires latest SimSecNoUI.exe
- Major cyber1 scene changes removing all the old stuff we don't need anymore
